src/styles/index.scss

```diff
@@ -1,3 +1,140 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
 }
+
+ul {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+ul li {
+  padding: 0;
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+  border-radius: 5px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  // &__img {
+  //   display: block;
+  //   width: 160px;
+  //   height: 134px;
+  //   background-image: url(../images/imac.jpeg);
+  //   background-position: center;
+  //   background-repeat: no-repeat;
+  //   background-size: contain;
+  //   bac
+  //   margin: 0 3px 40px;
+
+  //   // margin-bottom: 40px;
+  // }
+  &__img {
+    margin: 0 2px 0 3px;
+    img {
+      height: 134px;
+      width: 160px;
+
+      margin-bottom: 36px;
+    }
+  }
+
+  &__title {
+    color: #060b35;
```

Comment:
Also, you can create separate file with styles, and move all colors in variables, then import and use in all places of your project.